content/e/kpw/2025-04-11-kpw-arena-28.md

```diff
@@ -18,6 +18,9 @@ This show is set to feature two championship fights - a Street Fight between [Ch
 
 #### Build-up
 
+* On 4.04.2025 KPW [released a video][rosetti-zapowiada] in which the acting chairman [Rosetti](@/w/rosetti.md) stated that the promotion needs a full-time leader, and since [Krystian Malinowski](@/w/krystian-malinowski.md) still shows no signs of live, a new KPW chairman will be sworn in at the show (strongly implying himself).
```

Comment:
```suggestion
* On 4.04.2025 KPW [released a video][rosetti-zapowiada] in which the acting chairman [Rosetti](@/w/rosetti.md) stated that the promotion needs a full-time leader, and since [Krystian Malinowski](@/w/krystian-malinowski.md) still shows no signs of life, a new KPW chairman will be sworn in at the show (strongly implying himself).
```